docs/pages/components/step-input.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Step Input
+id: step-input
+keywords: Step Input, Step, Input
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/step-input.html
+folder: components
+---
+
+The Step Input component is an opinionated composition of the Input, Button components.
```

Comment:
Can we use the description from the Fiori Guidelines? I think it sounds better. I also included the usage guidelines in case you want to add them.

The step input control allows the user to change the input values in predefined increments (steps).

Usage
Use the step input if:
The user needs to adjust amounts, quantities, or other values quickly.
The user needs to adjust values for a specific step (for example, in a shopping cart).
Do not use the step input if:
The user needs to enter a static number (for example, postal code, phone number, or ID). In this case, use the regular input field control instead.
You want to display a value that rarely needs to be adjusted and does not pertain to a particular step. In this case, use the regular input field control instead.
You want the user to enter dates and times. In this case, use the date picker, date range selection, time picker, or date/time picker instead.